deployments/ansible/roles/collector/defaults/main.yml

```diff
@@ -28,8 +28,8 @@ splunk_service_user: splunk-otel-collector
 splunk_service_group: splunk-otel-collector
 
 splunk_memory_total_mib: 512
-# 1/3 of memory_mib by default
-splunk_ballast_size_mib: ""
+
+go_mem_limit: ""
```

Comment:
Nit: To be consistent with the variable name you added for puppet, should we change this and all other references to `gomemlimit`?

Reply:
The reason I have it this way because all the env var in this file has 'snake_cases'. I can change it if you like.

Reply:
removed.